fdbclient/RESTUtils.actor.cpp

```diff
@@ -30,6 +30,15 @@
 
 #include "flow/actorcompiler.h" // always the last include
 
+const std::string forwardSlash = "/";
+const std::string localhost = "0.0.0.0";
+
+namespace {
+bool isLocalConnection(const std::string& host) {
+	return host.compare("localhost") == 0 && host.compare(localhost) && host.compare("127.0.0.1");
```

Comment:
should these be `||` instead of `&&`?

Reply:
My bad should be ''||'.

Reply:
Good catch! Updated and re-tested the patch. Thanks